import-service/handlers/importFileParser.ts

```diff
@@ -0,0 +1,55 @@
+import * as AWS from 'aws-sdk';
+import * as csv from 'csv-parser';
+import { S3Handler } from 'aws-lambda';
+import 'source-map-support/register';
+
+import { BUCKET_NAME } from '../constants';
+
+const importFileParser: S3Handler = async (event) => {
+  try {
+    const s3 = new AWS.S3({
+      region: 'eu-west-1'
+    });
+
+    const record = event.Records[0];
+
+    const s3Object = record.s3.object;
+
+    const s3Stream = s3.getObject({
+      Bucket: BUCKET_NAME,
+      Key: s3Object.key,
+    }).createReadStream();
+
+    const results = [];
+
+    s3Stream
+      .pipe(csv({ separator: ';' }))
+      .on('data', (data) => {
+        console.log(data);
+
+        results.push(data)
+      })
+      .on('end', () => {
+        console.log(results);
+        console.log('end');
+      });
+
+    await s3.copyObject({
```

Comment:
tell me, if I'm wrong, but this code will not wait untill file is parsed.

Reply:
indeed, copying and deleting should be invoked in .on('end', () => {}) callback

Reply:
thanks for noticing, fixed